mahta/JS/JS basic-syntax/script.js

```diff
@@ -0,0 +1,100 @@
+function add(a,b){
+    return(a + b);
+}
+
+function flipACoin() {
+    var random = Math.random();
+    if (random > 0.5) {
+        retuen("tail");
+    } else if (random <= 0.5) {
+        return("head");
+    }
+}
+
+function concat(stringA, stringB) {
+    return(stringA + stringB);
+}
+
+function power2(x) {
+    return(Math.pow(2,x));
+}
+
+function getUser(firstName, lastName) {
+    const FirstName=firstName.replace(firstName.charAt(0),firstName.charAt(0).toUpperCase());
+    const LastName=lastName.replace(lastName.charAt(0),lastName.charAt(0).toUpperCase());
+    return{FirstName, LastName};
+}
+
+function greet(name) {
+    console.log(`Hi ${name}, nice to meet you!`)
+}
+
+function getNRandomNumbers(n) {
+    let random_numbers=[];
+    for (i = 0; i < n; i++){
+        random_numbers.push(Math.random()*1000);
+    }
+    return random_numbers;
+}
+
+function getNRandomNumbers(n, min, max) {
+    let random_numbers=[];
+    for (i = 0; i < n; i++){
+        random_numbers.push(Math.random()*(max-min)+min);
+    }
+    return random_numbers;   
+}
+
+//Not functioning
+function logCurrentTime(x, n) {
```

Comment:
write it using `setInterval`. If you struggle stopping the timer, google about clearing timeouts in js.

Reply:
I tried using it before, but it didn't function then as well. I'll work on it more tomorrow.